queries/d/highlights.scm

```diff
@@ -219,6 +200,35 @@
   "try"
 ] @exception
 
+"null" @constant.builtin
+
+[
+  "__gshared"
+  "const"
+  "immutable"
+  "shared"
+  "synchronized"
```

Comment:
`synchronized` is a keyword marking a class or a block as accessible by one thread at a time (https://tour.dlang.org/tour/en/multithreading/synchronization-sharing). I'm not sure which capture fits the best, maybe `@storageclass.lifetime`?